src/main/java/frc/robot/subsystems/Dashboard.java

```diff
@@ -0,0 +1,343 @@
+
+        _climberRightOffset = climbSetting.add("Right Offset", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        _climberMinHeight = climbSetting.add("Arm Minimum Height", 0.0).withPosition(0, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        _climberMaxHeight = climbSetting.add("Arm Maximum Height", 0.0).withPosition(2, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        var noteSetting = outerLayout.getLayout("Notepath", BuiltInLayouts.kList).withPosition(3, 0).withSize(8, 10).withProperties(Map.of("label position", "LEFT"));
+
+        _NoteShootSpeed = noteSetting.add("Note Shoot Speed", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        _PickupIntakeSpeed = noteSetting.add("Pickup Intake Speed", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        _ShooterIntakeSpeed = noteSetting.add("Shooter Intake Speed", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        var shooterBedSetting = outerLayout.getLayout("Shooter Bed", BuiltInLayouts.kList).withPosition(4, 0).withSize(7, 10).withProperties(Map.of("label position", "LEFT"));
+
+        _bedMinimumAngle = shooterBedSetting.add("Bed Minimum Angle", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _bedMaximumAngle = shooterBedSetting.add("Bed Maximum Angle", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _ShooterOffset   = shooterBedSetting.add("Shooter Relative Offset", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        var shooterFlywheelSetting = outerLayout.getLayout("Shooter Flywheel", BuiltInLayouts.kList).withPosition(5, 0).withSize(7, 10).withProperties(Map.of("label position", "LEFT"));
+
+        _maxFlywheelVelocity = shooterFlywheelSetting.add("Max Flywheel Velocity ", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _maxFlywheelSpeed    = shooterFlywheelSetting.add("Max Flywheel Speed ", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _FlywheelIntakeSpeed = shooterFlywheelSetting.add("Flywheel Intake Speed ", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        initializeSetting("FL Offset", Constants.Drive.MODULE_FL_OFFSET.getDegrees(), _flOffset, value ->
+        {
+            drive.setModuleAbsoluteEncoderOffset(0, Rotation2d.fromDegrees(value));
+        });
+
+        initializeSetting("FR Offset", Constants.Drive.MODULE_FR_OFFSET.getDegrees(), _frOffset, value ->
+        {
+            drive.setModuleAbsoluteEncoderOffset(1, Rotation2d.fromDegrees(value));
+        });
+
+        initializeSetting("BL Offset", Constants.Drive.MODULE_BL_OFFSET.getDegrees(), _blOffset, value ->
+        {
+            drive.setModuleAbsoluteEncoderOffset(2, Rotation2d.fromDegrees(value));
+        });
+
+        initializeSetting("BR Offset", Constants.Drive.MODULE_BR_OFFSET.getDegrees(), _brOffset, value ->
+        {
+            drive.setModuleAbsoluteEncoderOffset(3, Rotation2d.fromDegrees(value));
+        });
+
```

Comment:
Do you want to group these setting initializations to make them easier to read? Like swerve module offsets, intake settings, etc.